tests/tests.rs

```diff
@@ -1,22 +1,17 @@
-#[cfg(feature = "chan_select")]
 extern crate compiletest_rs as compiletest;
 
-#[cfg(feature = "chan_select")]
 use std::path::PathBuf;
 
-#[cfg(feature = "chan_select")]
 fn run_mode(mode: &'static str) {
     let mut config = compiletest::Config::default();
-    let cfg_mode = mode.parse().ok().expect("Invalid mode");
 
-    config.mode = cfg_mode;
+    config.mode = mode.parse().expect("Invalid mode");
     config.src_base = PathBuf::from(format!("tests/{}", mode));
-    config.target_rustcflags = Some("-L target/debug".to_string());
+    config.link_deps();
```

Comment:
Using `link_deps()` has a tendency to break on Windows, because of spaces in paths, so I'd be careful when using it :-)

Reply:
The tests stop working on my machine if I replace `config.link_deps()` with `config.target_rustcflags = Some("-L target/debug".to_string());`. Any idea why?